octoprint_octorelay/__init__.py

```diff
@@ -40,7 +40,136 @@ def __init__(self):
             self.model[index] = {}
 
     def get_settings_defaults(self):
-        return DEFAULT_SETTINGS
```

Comment:
Dear @jneilliii , do you know why this does not work well?
And even this:

```py
 def get_settings_defaults(self):
        return DEFAULT_SETTINGS.copy()
```

why it makes setting saving partial?
Why do I have to keep static object inside the method to make it work?
I almost broke my mind on it.

Reply:
I think I figured this out.
`.copy()` keeps the references to the nested objects, even they are not assigned to any const/vars.

<img width="1146" alt="Screenshot 2023-08-01 at 01 11 32" src="https://github.com/borisbu/OctoRelay/assets/13189514/f2d5196a-482d-4e77-9322-208270050d38">